kuma/banners/jinja2/banners/includes/developer-needs.html

```diff
@@ -1,16 +1,16 @@
 <div id="developer-needs" class="mdn-cta-content-container">
     <h2 class="mdn-cta-title">
-        {{ _('MDN Developer Survey') }}
+        {{ _('MDN survey') }}
```

Comment:
I'm not an English expert but I really don't like this lowercasing. I think it should be "MDN Survey" since it's a header. 
Actually, I checked with a close friend (British) who is really good at this and he too thinks it should be "MDN Survey". 

I don't know the background of where this came from but I would love to have one last look at it @atopal 

Also, @davidflanagan you're pretty good with words :) WHat do you think?

Reply:
Yeah, that's a typo. It was supposed to be "MDN Survey". Thanks for catching that, Peter.

Reply:
upated.